src/index.js

```diff
@@ -2,9 +2,12 @@ import React from 'react';
 import ReactDOM from 'react-dom';
 import Nav from './components/Nav';
 
-const app = document.createElement('app');
-ReactDOM.render(<Nav />, app);
+
+ReactDOM.render(<Nav />, document.getElementById('app')|| document.createElement('div'));
```

Comment:
this is great work but is the OR statement here necessary ?

Reply:
for me the OR is ok this index.js was causing error in two 2 ways ,
now it can take both